src/components/editions/header.tsx

```diff
@@ -12,60 +13,84 @@ import Series from 'components/editions/series';
 import Standfirst from 'components/editions/standfirst';
 import type { Item } from 'item';
 import type { FC, ReactElement } from 'react';
-
 // ----- Component ----- //
 
 interface Props {
 	item: Item;
 }
 
+interface HeaderProps {
+	item: Item;
+	className?: SerializedStyles;
+}
+
 const headerStyles = css`
+	padding: 0 ${remSpace[3]} ${remSpace[4]};
+
+	${from.desktop} {
+		padding: 0 ${remSpace[24]} ${remSpace[4]};
+	}
+`;
+
+const reviewHeaderStyles = css`
+	background-color: ${culture[800]};
+	color: ${culture[300]};
+`;
+
+const showcaseHeaderStyles = css`
+	padding: 0 ${remSpace[2]};
 	margin: 0;
 
 	${from.wide} {
 		margin: 0 auto;
 	}
 
 	${from.phablet} {
+		padding-left: 0;
+		padding-right: 0;
 		margin-left: ${remSpace[24]};
 	}
 `;
 
-const StandardHeader: FC<Props> = ({ item }) => (
-	<>
+const StandardHeader: FC<HeaderProps> = ({ item, className }) => (
+	<header css={[headerStyles, className]}>
 		<HeaderImage item={item} />
 		<Series item={item} />
 		<Headline item={item} />
 		<Standfirst item={item} />
 		<Lines />
 		<Byline item={item} />
-	</>
+	</header>
 );
 
-const ShowcaseHeader: FC<Props> = ({ item }) => (
-	<>
+const ShowcaseHeader: FC<HeaderProps> = ({ item }) => (
+	<header css={showcaseHeaderStyles}>
 		<Series item={item} />
 		<Headline item={item} />
 		<HeaderImage item={item} />
 		<Standfirst item={item} />
 		<Lines />
 		<Byline item={item} />
-	</>
+	</header>
 );
 
 const renderArticleHeader = (item: Item): ReactElement<Props> => {
-	switch (item.display) {
-		case Display.Showcase:
-			return <ShowcaseHeader item={item} />;
+	if (Display.Showcase === item.display)
+		return <ShowcaseHeader item={item} />;
+	switch (item.display | item.design) {
```

Comment:
Does it make sense to be more implicit here i.e. 
```
if(item.design === Design.Review){}
```
Do we gain anything from the `switch` as it is?

Reply:
Good point - I'll edit in favour of if/else here